### PR TITLE
Summary page

### PR DESCRIPTION
**Pull request by mike-suggitt**

### CODE REVIEW

File: src/app/case.service.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export class CaseService {
   constructor(private httpClient: HttpClient) { }
 
 
-  fetch(): Observable<Object> {
-        return this.httpClient.get('http://localhost:3000/api/cases/bob');
+  fetch(caseId): Observable<Object> {
+        return this.httpClient.get(`http://localhost:3000/api/cases/${caseId}`);
```
Comment (Louisblack) [inline src/app/case.service.ts]: Can this be relative? So it works deployed.

**Comment by mike-suggitt** [comment]
Yea that's fair enough. Can I politely request 4 spaces for indents? It's
pretty much the only limiting I care about as I can't read 2 space code!

**Comment by Louisblack** [comment]
4 spaces have my vote. I know 2 is the kinda new thing but I agree 4 is more readable.

File: src/app/case.service.ts (hunk 1 above)
Comment (mike-suggitt) [reply src/app/case.service.ts]: Yes definitely but would like to do it as a dedicated piece of work that thinks about environments and universal data fetching

**Pull request merged**